### PR TITLE
Iter2

### PR DESCRIPTION
**Pull request by Quarol**

### CODE REVIEW

File: src/Calibration.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+interface EyeTrackerProps {
+    onFinishCalibration: () => void;
+  }
+
+const Calibration: React.FC<EyeTrackerProps> = ({ onFinishCalibration }) => {
+    const [clickCount, setClickCount] = useState([0, 0, 0, 0, 0]);
+    const [totalClicks, setTotalClicks] = useState(0);
+    const NUMBER_OF_DOTS = 5;
+    const MAX_CLICKS = 12;
```
Comment (mekiert-ocado) [inline src/Calibration.tsx]: [optional] The constants could be outside the component.

File: src/Calibration.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    const MAX_TOTAL_CLICKS = MAX_CLICKS * NUMBER_OF_DOTS;
+
+
+    const handleClick = (index: number) => {
+        if (clickCount[index] < MAX_CLICKS) {
+            const newClickCount = [...clickCount];
+            newClickCount[index]++;
```
Comment (mekiert-ocado) [inline src/Calibration.tsx:20]: The opration is modyfing the array. You might use a map here to store clicks.

```
const newClickCount = {...clickCount, [index]: clickCount[index]++}
```

File: src/RotateObject3D.tsx [hunk 3]
```diff
@@ -49,15 +50,24 @@ const RotateObject3D = () => {
...
+      setRotationSpeed();
     }
 
-    document.addEventListener('mousemove', handleMouseMove);
+    document.addEventListener('mousemove', handleMovement);
+    */
+
+    function setRotationSpeed() {
+      if (Coordinates.isSet) {
+        //const mouseX = (event.clientX / window.innerWidth) * mouseXScaleFactor + mouseXOset;
```
Comment (mekiert-ocado) [inline src/RotateObject3D.tsx]: Please remove useless comment. If it is code that will be used in the future, then describe it appropriately.

**Review by mekiert-ocado — CHANGES_REQUESTED** [review]

File: src/Calibration.tsx (hunk 2 above)
Comment (mekiert-ocado) [reply src/Calibration.tsx:20]: @Quarol It is still not resolved.

File: src/Calibration.tsx (hunk 1 above)
Comment (mekiert-ocado) [reply src/Calibration.tsx]: @Quarol It is still not resolved.

File: src/RotateObject3D.tsx (hunk 3 above)
Comment (mekiert-ocado) [reply src/RotateObject3D.tsx]: @Quarol  It is still not resolved.

**Comment by Quarol** [comment]
Latest changes were not synced with pull request, I have to create a new pull request.

**Pull request closed without merging**